deployments/clowdapp.yml

```diff
@@ -82,6 +82,24 @@ objects:
         image: ${IMAGE}:${IMAGE_TAG}
         command: 
           - ./pt-consumer
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /live
+            port: 8001
+            scheme: HTTP
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /ready
+            port: 8001
```

Comment:
Is this the right port?  I _think_ the metrics port will get configured by clowder to be 9000?